internal/constants/scripts.go

```diff
@@ -133,3 +133,80 @@ function correlate_int_mac() {
 
 correlate_int_mac
 `
+
+// PreNetworkManagerConfigScript script runs on hosts before network manager service starts in order to copy
+// the host specifics *.nmconnection files to /NetworkManager/system-connections/ based on host MAC-address matching
+const PreNetworkManagerConfigScript = `
+#!/bin/bash
+
+#TODO: set file path for unified nmconnection file of static conf
```

Comment:
@yevgeny-shnaidman  what is the expected filepath for the unified nmconnection file ?

Reply:
let's put them under /etc.